umh.docs.umh.app/content/en/docs/production-guide/administration/access-services-from-outside-cluster.md

```diff
@@ -49,20 +49,21 @@ To access Node-RED, you need to use the `/nodered` path, for example
 
 ## Services with NodePort by default
 
-The [Kafka Broker](/docs/architecture/microservices/core/kafka-broker/) uses the service type NodePort by default. 
+The [Kafka Broker](/docs/reference/microservices/kafka-broker/) uses the service 
+type NodePort by default. 
 
 Follow these steps to access the Kafka Broker outside the cluster:
 
 1. Access your instance via SSH
-2. Execute this command:
+2. Execute this command to check the host port of the Kafka Broker:
 
    ```bash
-   sudo $(which helm) upgrade --set redpanda.external.type=LoadBalancer united-manufacturing-hub united-manufacturing-hub/united-manufacturing-hub -n united-manufacturing-hub --reuse-values --version $(sudo $(which helm) get metadata united-manufacturing-hub -n united-manufacturing-hub --kubeconfig /etc/rancher/k3s/k3s.yaml -o json | jq '.version') --kubeconfig /etc/rancher/k3s/k3s.yaml
+   sudo $(which kubectl) get svc united-manufacturing-hub-kafka-external -n united-manufacturing-hub --kubeconfig /etc/rancher/k3s/k3s.yaml
    ```
 
-3. Wait for the changes to be applied.
+3. In the `PORT(S)` column, you should be able to see the port with `9094:<host-port>/TCP`.
+4. To access the Kafka Broker, use `<instance-ip-address>:<host-port>`.
 
-Access the Kafka Broker at port 9094.
 
 ## Services with ClusterIP
 
```

Comment:
I would explain here the two different options, create a loadbalancer or port forward

Reply:
Added explanation